package.json

```
@@ -14,7 +14,7 @@
     "test6:sanity": "lerna run --scope wix-style-react test6:sanity",
     "test7:performance": "lerna run --scope wix-style-react test7:performance",
     "test8:chromatic": "lerna run --scope wix-style-react-test-runtime test",
-    "test9": "lerna run build --scope wix-ui-tpa && lerna run test --scope wix-ui-tpa",
+    "test9": "lerna run build --scope wix-ui-tpa && lerna run test --scope wix-ui-tpa && lerna run storybook:release --scope wix-ui-tpa",
```

```suggestion
    "test9": "lerna run build --scope wix-ui-tpa && lerna run storybook:release --scope wix-ui-tpa && lerna run test --scope wix-ui-tpa",
```

i'd think maybe if tests fail then storybook is falsy and shouldn't be published?
but if that's ok, then you can commit this suggestion and merge

---

but that's what happens with wsr as well. Janet is parallel